forge-game/src/main/java/forge/game/ability/effects/AnimateEffectBase.java

```diff
@@ -183,11 +210,17 @@ public String getDescription() {
         if (removeAll || removeNonManaAbilities
                 || !addedAbilities.isEmpty() || !removedAbilities.isEmpty() || !addedTriggers.isEmpty()
                 || !addedReplacements.isEmpty() || !addedStaticAbilities.isEmpty()) {
-            c.addChangedCardTraits(addedAbilities, removedAbilities, addedTriggers, addedReplacements,
-                    addedStaticAbilities, removeAll, removeNonManaAbilities, timestamp, 0);
+            final CardTraitChanges ctc = new CardTraitChanges(addedAbilities, removedAbilities, addedTriggers, 
+                addedReplacements, addedStaticAbilities, removeAll, removeNonManaAbilities);
+            c.addChangedCardTraits(ctc, timestamp, 0);
```

Comment:
it's debatable if storing the ctc too would improve accuracy
otherwise this can be cleaned up again